frb_codegen/src/library/codegen/polisher/mod.rs

```diff
@@ -109,11 +109,14 @@ fn execute_build_runner(
 }
 
 fn execute_dart_fix(
-    config: &PolisherInternalConfig,
+    output_paths: &[PathBuf],
     progress_bar_pack: &GeneratorProgressBarPack,
 ) -> anyhow::Result<()> {
     let _pb = progress_bar_pack.polish_dart_fix.start();
-    dart_fix(&config.dart_root)
+    for output_path in output_paths {
```

Comment:
nit: it may be too slow because IIRC output_paths are concrete .dart files instead of a folder, and doing tens of `dart fix` may not be very fast

Reply:
Yeah I was testing those parameters. 😄